{"timestamp": "2025-02-18-12-15-05", "message": "Module error while trying to update local copy of weekly_cycle/1: failed to download sheet '<googleapiclient.discovery.Resource object at 0xf468eb10>' from spreadsheet 'f\u0171t\u00e9s heti ciklus 24/25': <HttpError 503 when requesting https://sheets.googleapis.com/v4/spreadsheets/1OmcLOB4OvFqmVw6Jq1lxu5fo7FchPBNMM6MFoNgM7r4/values/Oktopusz%21A1%3AZ10000?alt=json returned \"The service is currently unavailable.\". Details: \"The service is currently unavailable.\"> (HttpError).", "severity": 3, "origin": "schedule_and_config_updater.py/update_local_scheduling_files:173 --> project.py/download_google_sheet_to_2D_array:1744", "origin_timestamp": "2025-02-18-12-13-56", "registration_timestamp": "2025-02-18-12-13-56"}
{"timestamp": "2025-02-18-18-20-08", "message": "Module error while trying to update local copy of weekly_cycle/1: failed to set up Google Sheets API: The read operation timed out (TimeoutError).", "severity": 3, "origin": "schedule_and_config_updater.py/update_local_scheduling_files:173 --> project.py/download_google_sheet_to_2D_array:1722", "origin_timestamp": "2025-02-18-18-18-35", "registration_timestamp": "2025-02-18-18-18-35"}
